sql/updates/world/3.3.5/9999_99_99_99_world_CrownChemical.sql

```diff
@@ -0,0 +1,234 @@
+SET @CGUID := 400000;
+SET @OGUID := 400000;
+
+UPDATE `creature_template` SET `unit_flags`=33280, `ScriptName`='boss_apothecary_hummel' WHERE `entry`=36296;
+UPDATE `creature_template` SET `unit_flags`=33280, `ScriptName`='npc_apothecary_baxter' WHERE  `entry`=36565;
+UPDATE `creature_template` SET `unit_flags`=33280, `ScriptName`='npc_apothecary_frye' WHERE  `entry`=36272;
+UPDATE `creature_template` SET `unit_flags`=33555200 WHERE  `entry` IN(36710,36530);
+UPDATE `gameobject_loot_template` SET `QuestRequired`=0 WHERE `Entry`=28677 and`Item`=49352;
```

Comment:
QuestRequired = true -> check where you can add that it should track the quest related to the LFGEncounter so that the items can be looted if the quest is active